src/sentry_log.h

```diff
@@ -0,0 +1,15 @@
+#include "sentry.h"
+
+void sentry_log_hook_default(
+    sentry_level_t level, const char *message, va_list args);
+
+void sentry_log(sentry_level_t level, const char *message, ...);
+
+#define SENTRY_DEBUGF(message, ...)                                            \
+    sentry_log(SENTRY_LEVEL_DEBUG, message "\n", __VA_ARGS__)
+
+#define SENTRY_DEBUG(message) sentry_log(SENTRY_LEVEL_DEBUG, message "\n")
+
+// for compatibility with existent macros
+#define SENTRY_TRACE SENTRY_DEBUG
```

Comment:
hm, I’m wondering if we would want to actually make use of more log levels. Would be nice to actually have proper `WARN` there, and have everything else rather as `INFO`, and trace as `DEBUG`. But that should be a followup.

Reply:
added `SENTRY_WARN` and output severity level in default logger.